pontoon/base/views.py

```diff
@@ -784,6 +785,53 @@ def translation_memory(request):
         return HttpResponse("no")
 
 
+def translation_memory_pg(request):
+    """
+    Suggests translations from internal translations.
+    """
+    try:
+        text = request.GET['text']
+        locale = request.GET['locale']
+        pk = request.GET['pk']
+    except MultiValueDictKeyError as e:
+        log.error(str(e))
+        return HttpResponse("error")
```

Comment:
You'll want to use HttpResponse subclasses that have appropriate HTTP codes for the error.